src/main/java/com/mailjet/client/transactional/response/EmailResult.java

```diff
@@ -0,0 +1,26 @@
+package com.mailjet.client.transactional.response;
+
+import lombok.Data;
+
+@Data
+public class EmailResult {
+    /**
+     * The email address of this recipient.
+     */
+    private String email;
+
+    /**
+     * A 128-bit universally unique identifier (UUID) for this message.
+     */
+    private String messageUUID;
+
+    /**
+     * Unique numeric ID of this message.
+     */
+    private long messageID;
```

Comment:
would be good to describe how this differs from the messageUUID? Isn't either of the two legacy?

Reply:
The difference is obvious - one is numeric, whereas the other is UUID :)

Reply:
I was thinking about it from a user's perspective. Let's say I want to store the ID of every sent message. Which one would I then take? Are they both the same, but in a different format? If so, why are both of them here?

Wasn't one of them a legacy identifier? (I thought I read that somewhere else some time ago)

Reply:
Yep, I got your direction of thoughts :)

Probably, more logical to use numeric id, as you can use it to get the message status through /message/ rest endpoint. Ill update the comment.

Reply:
updated XML doc